### DISCUSSION TITLE
how to export apk from flet app?

### DISCUSSION THREAD

**Discussion by hossainirad** [flet-dev/flet#167] [General]
Hi every one.
How to export apk from flet app for android mobile application purpose?

Thank you.

**Comment by Shahin-rmz** [comment]
the Framework seems **very** cool though.

**Comment by FeodorFitsner** [comment]
So, we put the plan here: https://flet.dev/blog/flet-mobile-strategy. In short, building a standalone mobile package (.apk, .ipa) is in our todo list.

Sorry, we should have added it to the docs for better visibility. Let me know if you have any questions.

**Reply by hossainirad** [reply]
Thank you so  much. Do you have any idea about flet mobile export optimization?
I am concern about optimization might be like Kiwi.

**Comment by hossainirad** [comment]
Still can't get apk output?

**Reply by ndonkoHenri** [reply]
Not yet.

**Comment by liudonghua123** [comment]
Is there any updates about produce apk, ipa for android, ios? I saw https://flet.dev/blog/flet-for-android, https://flet.dev/blog/flet-for-ios, but could not find a way to go.
